src/transformations.ts

```diff
@@ -103,6 +103,13 @@ export const toStrings = (
       const locale = pod.locale(localeId);
       let value = column.trim();
 
+      // Fixes an issue where the Sheets API removes a single quote from the 
+      // start of the string. 
+      // https://github.com/blinkk/amagaki-plugin-google-sheets/issues/22
+      if (value.endsWith("'") && !value.startsWith("'")) {
```

Comment:
https://docs.google.com/spreadsheets/d/1McQm0kF96i67ZIYtIcqlFAOCOhDh14XQ4DN8m6NGv3I/edit?resourcekey=0-93Ku3tclwc1xzc8vdJjKGQ#gid=0&range=J94

Agree with your points.

Here's an example cell. I think the issue is that an apostrophe at the beginning of the contents has special significance in a spreadsheet

Reply:
OK... I learned a bit more about apostrophes and Google Sheets and Excel, too. This looks like a feature of Google Sheets like you were saying; it is used to indicate to Sheets that the contents should be treated like a string (and not a number or some other value).

https://superuser.com/questions/806771/why-would-all-text-columns-in-an-excel-spreadsheet-begin-with-a-leading-single-q

Given this is purposeful behavior I think the right (but unfortunate) fix here is to modify the spreadsheet contents:

- Either add two apostrophes
- Use double quotes instead of single quotes in the sheet

For the sheet you listed I think the best approach should be to just replace it with two apostrophes. You'll notice that the "preview" in Google Sheets also omits the apostrophe (so it's not just the API response). Is that OK?